deploy/installer_machine/prepare_installer.tpl

```diff
@@ -2,19 +2,21 @@
 exec > >(tee /var/log/user-data.log|logger -t user-data -s 2>/dev/console) 2>&1
 set -e
 
-sudo yum -y install git
+sudo su
+yum -y install git
 
-sudo yum install -y yum-utils
-sudo yum-config-manager --add-repo https://rpm.releases.hashicorp.com/RHEL/hashicorp.repo
-sudo yum -y install terraform
-sudo yum -y install java
+yum install -y yum-utils
+yum-config-manager --add-repo https://rpm.releases.hashicorp.com/RHEL/hashicorp.repo
+yum -y install terraform
+yum -y install java-11-openjdk-devel-11.0.17.0.8-2.el8_6
```

Comment:
I think this is too specific and might break if they release a new redhat. I would try something like
> yum install java-11-openjdk-devel